### ISSUE TITLE
Enhance Multipart scanning UX

### ISSUE THREAD

**Issue by Tbaut** [paritytech/parity-signer#426]
When I scan a big multipart QR code my phone might miss some of the codes.
If I scan say a 10 parts code, I miss the 10th:
- I have no idea that I missed it if I don't carefully keep track of the QR codes changes and the number being iterated
- I have no idea which one I missed, and I'll need to remain steady for another full round of QR hoping not to miss it another time
- I have to wait (at least) one other full round (this is on the code generation part)

TL;DR
I'd be nice to know if we missed a QR code, and get some visualization of where we are in the scanning process represented by boxes white (not yet scanned), green (scanned successfully) or red (missed) e.g:
. . :white_check_mark: :white_check_mark: :stop_sign: :white_check_mark: . . .

**Comment by yjkimjunior** [comment]
#320 related
